hack/chains/rekor-verify-taskrun.sh

```diff
@@ -8,6 +8,7 @@ TASKRUN_NAME=${1:-$( tkn taskrun describe --last -o name )}
 TASKRUN_NAME=taskrun/$( trim-name $TASKRUN_NAME )
 
 # Let's not hard code the image url or the registry
+IMAGE_DIGEST=$( kubectl get $TASKRUN_NAME -o json | jq -r '.status.taskResults[0].value' )
```

Comment:
It might be better to not rely on the order of the results. How about something like this:
```
jq -r '.status.taskResults[] | select(.name=="IMAGE_DIGEST") | .value'
```

We could do something similar on the line below.

Reply:
That's a good suggestion. Let me fix it.

Reply:
https://github.com/simonbaird/infra-deployments/blob/c5b101b26e0476529fd36f140abb4f5b904a772e/hack/contract/fetch-data.sh#L42 is relevant.